src/collar/oc_sys.lsl

```diff
@@ -687,6 +687,7 @@ default {
 
     on_rez(integer iParam) {
         g_iHide=!(integer)llGetAlpha(ALL_SIDES) ; //check alpha
+        llSleep(10.0);
```

Comment:
I think you have this backwards. If oc_sys sleeps for 10 and settings sleeps for 7, sys will not hear the settings.

Reply:
it is the same order that can be seen in the state_entry. Maybe we should let both sleep for 10 sec?

Reply:
Still wouldn't make a lot of sense to do that. Since we need all scripts to hear settings

Reply:
so the other way arround? like wait for 10 seconds in oc_settings and 7 seconds in oc_sys?

Reply:
That may be better than how it presently is yes